src/components/result-ui/display-raw.js

```diff
@@ -23,16 +27,19 @@ const DisplayRaw = ({ title, name, responseDocument, children }) => {
             <TabMenu id={1} title="Raw" />
           </ul>
           <div className={`tab flex-height ${activeTab === 0 ? 'tab__active' : ''}`}>
-            <h2 className="tab__title">{title}</h2>
-            {(!Array.isArray(responseDocument.data) || responseDocument.data.length)
-              ? children
-              : <div />
-            }
+            <div className="tab__header">
+              <span className="tab__header__result_count">{isCollection ? resultCount ? `${resultCount} results` : 'No results' : 'Result'}</span>
```

Comment:
Much better to use a responseDocument method here! Also glad you added a no results message at the same time.